kzz7yh-f58197/cod-ve09v2_kzz7yh-f58197.xml

```diff
@@ -280,9 +283,10 @@
       forti<lb ed="#V" n="59" break="no"/>us modo mouentur motu delectationis in actu carnalis 
       co<lb ed="#V" n="60" break="no"/>pule: quam in statu innocentie: sicut equus lasciuus rupto freno 
       <lb ed="#V" n="61"/>fortius mouetur quam equus fortior cum frenatur et tenetur a suo sessore. 
-      <lb ed="#V" n="62"/>Qui vult tenere primam opi. quie multis videtur 
-      probabi<lb ed="#V" n="63" break="no"/>lior potest dicere ad principium in oppsium quod 
-      ob<lb ed="#V" n="64" break="no"/>nubiltio rationis modo non est propter intentionem delectationis tantum. Sed 
+      <!--TODO Add paragraph-->
+      <lb ed="#V" n="62"/>Qui vult tenere primam opi. qui multis videtur 
```

Comment:
quae not qui since it refers to the feminine "operationem" and I think we can expand the abbreviation for "opinionem"

```suggestion
      <lb ed="#V" n="62"/>Qui vult tenere primam opinionem quae multis videtur 
```